src/App.tsx

```diff
@@ -5,7 +5,9 @@ import {
   Route
 } from 'react-router-dom';
 import { Home } from './Home/Home';
+import { Curiosity} from './Curiosity/curiosity';
```

Comment:
Let's try and have consistent spacing around our curly braces for imports so just have one either side for here and the Spirit import :)